modules/olm-overriding-proxy-settings.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-adding-operators-to-cluster.adoc
+
+[id="olm-overriding-proxy-settings_{context}"]
+= Overriding an Operator's proxy settings
+
+If a cluster-wide egress proxy is configured, applications created from
+Operators using the Operator Lifecycle Manager (OLM) inherit the cluster-wide
+proxy settings on their Deployments and Pods. Cluster administrators can also
+override these proxy settings by configuring the Operator's Subscription.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+`cluster-admin` permissions.
+endif::[]
+ifdef::openshift-dedicated[]
+`dedicated-admins-cluster` permissions.
+endif::[]
+
+.Procedure
+
+. Navigate in the web console to the *Operators → OperatorHub* page.
+
+. Select the Operator and click *Install*.
+
+. On the *Create Operator Subscription* page, modify the Subscription object's
+YAML to include one or more of the following environment variables in the
+`spec` section:
++
+--
+* `HTTP_PROXY`
+* `HTTPS_PROXY`
+* `NO_PROXY`
+--
++
+For example:
++
+.Subscription object with proxy setting overrides
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: etcd-config-test
+  namespace: openshift-operators
+spec:
+  config:
+    env:
+    - name: HTTP_PROXY
+      value: test_http
+    - name: HTTPS_PROXY
+      value: test_https
+    - name: NO_PROXY
+      value: test
+  channel: clusterwide-alpha
+  installPlanApproval: Automatic
+  name: etcd
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: etcdoperator.v0.9.4-clusterwide
+----
++
+[NOTE]
+====
+These environment variables can also be unset using an empty value to remove any
+previously set cluster-wide or custom proxy settings.
```

Comment:
This note will go into master and 4.4, but will be swapped in 4.2 and 4.3 for info on the Known Issue https://bugzilla.redhat.com/show_bug.cgi?id=1751903, with a link to the relevant release note.